examples/oft-solana/README.md

```diff
@@ -34,6 +36,41 @@ pnpm install
 pnpm test
 ```
 
+### Get Devnet SOL
+
+```bash
+solana airdrop 5 -u devnet
+```
+
+We recommend that you request 5 devnet SOL, which should be sufficient for this walkthrough. For the example here, we will be using Solana Devnet. If you are hitting rate limits after, you can also use the [official Solana faucet](https://faucet.solana.com/).
+
+### Prepare `.env`
+
+```bash
+cp .env.example .env
+```
+
+In the `.env` just created, set `SOLANA_PRIVATE_KEY` to your private key value in base58 format. Since the locally stored keypair is in an integer array format, we'd need to encode it into base58 first. You can create a temporary script called `getBase58Pk.js` in your project root with the following contents:
+
+```js
+import fs from "fs";
+import { Keypair } from "@solana/web3.js";
+import bs58 from "bs58";
+
+const keypairFilePath = `<KEYPAIR_FILE_PATH_HERE>`; // you can view this by running `solana config get`
+
+const data = fs.readFileSync(keypairFilePath, "utf8");
+const keypairJson = JSON.parse(data);
+const keypair = Keypair.fromSecretKey(Uint8Array.from(keypairJson));
+const base58EncodedPrivateKey = bs58.encode(keypair.secretKey);
+
+console.log(base58EncodedPrivateKey);
+```
+
+Then, run `node getBase58Pk.js`
+
+Also set the `RPC_URL_SOLANA_TESTNET` value. Note that while the naming used here is `TESTNET`, it refers to the Solana Devnet. We use `TESTNET` to keep it consistent with the existing EVM testnets.
+
```

Comment:
You can link to the Solana Devnet link under "Chains" in the documentation here.